source/howto/pipelines/triggers.rst

```diff
@@ -11,17 +11,26 @@ Valohai triggers allow you to launch a copy of an existing pipeline on a time-ba
 .. note::
 
     You can create a trigger after you've ran at least one succesfully completed pipeline.
+    You can either schedule a specific (commit) version of a pipeline to run or schedule a pipeline run that always uses the latest version of the pipeline.
 
 ..
 
 1. Go to project settings
 2. Navigate to the Triggers tab
-3. Set your trigger title (e.g. *Weekly re-training*)
-4. Select *Scheduled (Cron)* as the condition, as click *Add*
-5. Open the condition and set the schedule you want. For example, Weekly on Mondays at 09:00
-6. On the *Actions* column select *Copy Pipeline* and click *Add*
-7. Open the action and select the *Source Pipeline* that you want to run
-8. Click on *Create trigger*
+3. Set your trigger title (e.g. Daily retraining, every 4 hours)
+4. Select Scheduled (Cron) as the condition, as click Add
+5. Open the condition and set the schedule you want.
+  ●	You can either use one of the options or select Custom
```

Comment:
These round marks don't seem to work. You can check either from other docs pages or from Sphinx documentation how to make lists.